From: Gabriel Filion <gabster@lelutin.ca>
Date: Sat, 10 Aug 2024 12:42:15 -0700
Subject: Don't build js libraries from third parties

Forwarded: not-needed
Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)

These 3rd party libraries are getting excluded from the source through
repacking since we instead include those from the debian packages that already
exists for them. The build shouldn't try and process the non-existent files.
---
 htdocs/Makefile.am | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/htdocs/Makefile.am b/htdocs/Makefile.am
index 27b9210..f626ae4 100644
--- a/htdocs/Makefile.am
+++ b/htdocs/Makefile.am
@@ -16,21 +16,6 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
 JS = js/smokeping.js \
-    js/prototype.js \
-    js/cropper/cropper.js \
-    js/cropper/licence.txt \
-    js/cropper/marqueeVert.gif \
-    js/cropper/cropper.uncompressed.js \
-    js/cropper/cropper.css \
-    js/cropper/marqueeHoriz.gif \
-    js/scriptaculous/controls.js \
-    js/scriptaculous/slider.js \
-    js/scriptaculous/sound.js \
-    js/scriptaculous/effects.js \
-    js/scriptaculous/unittest.js \
-    js/scriptaculous/builder.js \
-    js/scriptaculous/scriptaculous.js \
-    js/scriptaculous/dragdrop.js \
     css/smokeping-print.css \
     css/smokeping-screen.css
 
